### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by Canel420**
Adding new methods to the storage engines (Filestorage and DBstorage)
Methods:
- get (retrieve a specific object from storage)
- count (count the number of objects in storage)

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -64,6 +64,16 @@ def delete(self, obj=None):
         if obj is not None:
             self.__session.delete(obj)
 
+    def get(self, cls, id):
+        """ A method to retrieve one object """
+        return self.__session.query(cls).filter(cls.id == id).first()
+
+    def count(self, cls=None):
+        """ A method to count the number of objects in storage """
+        if cls is None:
```
Comment (miguecoder) [inline models/engine/db_storage.py:73]: is the same "cls is None" to "!cls!?

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -48,14 +48,27 @@ def save(self):
...
+        for object in self.__objects.values():
+            if object.__class__ == cls and object.id == id:
+                return object
+            return None
+
+    def count(self, cls=None):
+        """ A method to count the number of objects in storage """
+        if cls is None:
+            return len(self.all())
+        return len(self.all(cls))
```
Comment (miguecoder) [inline models/engine/file_storage.py:62]: What if don't exist the class?

**Review by miguecoder — APPROVED** [review]

File: models/engine/db_storage.py (hunk 1 above)
Comment (Canel420) [reply models/engine/db_storage.py:73]: I believe is different. None is a data type, and using ! Would mean that the class is false or zero

File: test_get_count.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
+""" Test .get() and .count() methods
+"""
+from models import storage
```
Comment (miguecoder) [inline test_get_count.py:4]: Missing a space here.

**Review by miguecoder — APPROVED** [review]

File: models/engine/file_storage.py (hunk 2 above)
Comment (Canel420) [reply models/engine/file_storage.py:62]: If the class doesn't exist the second return would tell us. Because it won't be able to find it

**Review by miguecoder — APPROVED** [review]

**Pull request closed without merging**